src/Symfony/Component/Process/PhpProcess.php

```diff
@@ -55,6 +55,9 @@ public static function fromShellCommandline(string $command, ?string $cwd = null
         throw new LogicException(\sprintf('The "%s()" method cannot be called when using "%s".', __METHOD__, self::class));
     }
 
+    /**
+     * @param (callable('out'|'err', string):bool)|null $callback
```

Comment:
to me the return value is never used, so this is fine without a return type (same everwhere)
```suggestion
     * @param callable('out'|'err', string)|null $callback
```